cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/entities/TerraformOrganizationInputs.java

```diff
@@ -0,0 +1,161 @@
+ /*
+  * (c) Copyright 2020 Micro Focus, L.P.
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+  *
+  * The Apache License is available at
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+ package io.cloudslang.content.hashicorp.terraform.entities;
+
+ import org.jetbrains.annotations.NotNull;
+
+ public class TerraformOrganizationInputs {
+
+
+     //        private final String organizationName;
+     private final String organizationDescription;
+     private final String email;
+     private final String sessionTimeout;
+     private final String sessionRemember;
+     private final String collaboratorAuthPolicy;
+     private final String costEstimationEnabled;
+     private final String ownersTeamSamlRoleId;
+     private final TerraformCommonInputs commonInputs;
+
+     @java.beans.ConstructorProperties({"organizationDescription", "adminEmail", "sessionTimeout", "sessionRemember", "collaboratorAuthPolicy", "costEstimationEnabled", "ownersTeamSamlRoleId",
```

Comment:
Be consistent with variable **email** or **adminEmail** use either of one

Reply:
Changes done.